extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -309,4 +332,58 @@ public void onLightAdded(HueBridge bridge, FullLight light) {
         }
     }
 
+    /**
+     * Schedules restoration of the alert item state to {@link LightStateConverter#ALERT_MODE_NONE} after a given time.<br>
+     * Based on the initial command:
+     * <ul>
+     * <li>For {@link LightStateConverter#ALERT_MODE_SELECT} restoration will be triggered after <strong>2
+     * seconds</strong>.
+     * <li>For {@link LightStateConverter#ALERT_MODE_LONG_SELECT} restoration will be triggered after <strong>15
+     * seconds</strong>.
+     * </ul>
+     * 
+     * @param channelUID The {@link ChannelUID} which received the initial command
+     * @param command The {@link Command} sent to the item
+     */
+    private void scheduleAlertStateRestore(final ChannelUID channelUID, Command command) {
+        int delay = getAlertDuration(command);
+
+        if (delay > 0) {
+            scheduler.schedule(new Runnable() {
```

Comment:
Yo are right, we have to cancel the `ScheduledFuture` also if new SELECT or LSELECT command is received in `handleCommand()` method, because the blinking effect could be restarted for 15 seconds on the 10th second after the first LSELECT, therefore the state will become NONE 10 seconds before the lamp stops the blinking effect.